### PR TITLE
Playpen Sans: Version 1.000 added

### PR DESCRIPTION
**Pull request by vv-monsalve**
896f02c: [gftools-packager] Playpen Sans: Version 1.000 added

* Playpen Sans Version 1.000 taken from the upstream repo https://github.com/TypeTogether/Playpen-Sans at commit https://github.com/TypeTogether/Playpen-Sans/commit/114342f694edaa1f63a51cbb216a10120fc271b5.

### CODE REVIEW

**Comment by vv-monsalve** [comment]
Before merging, we await confirmation on the order in which the designers should appear + biographical information.

**Comment by vv-monsalve** [comment]
@RosaWagner I was still creating the PR and I was about to add the related issue before you did it.

**Comment by RosaWagner** [comment]
Call me Flash-rosie

**Comment by vv-monsalve** [comment]
All right, everything is set up now 😎

File: ofl/playpensans/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+subsets: "vietnamese"
+axes {
+  tag: "wght"
+  min_value: 100.0
+  max_value: 800.0
+}
+source {
+  repository_url: "https://github.com/TypeTogether/Playpen-Sans"
+  commit: "114342f694edaa1f63a51cbb216a10120fc271b5"
+}
```
Comment (RosaWagner) [inline ofl/playpensans/METADATA.pb:27]: shall we already add the new categorisation stuff?

**Review by RosaWagner — APPROVED** [review]

File: ofl/playpensans/METADATA.pb (hunk 1 above)
Comment (RosaWagner) [reply ofl/playpensans/METADATA.pb:27]: will do in another PR

**Comment by RosaWagner** [comment]
Blocking this until the subset is merge internally (cf https://github.com/googlefonts/glyphsets/pull/127), and should then be repushed to_sandbox.

**Comment by chrissimpkins** [comment]
@RosaWagner @vv-monsalve OK to push a new PR **with the updated subsetter definitions in METADATA.pb**, merge, and push to sandbox.  Nate updated the subsetter defs yesterday.

**Pull request merged**